### DISCUSSION TITLE
Custom name for "Undated" column,

### DISCUSSION THREAD

**Discussion by ItsFullOfCode** [roovo/obsidian-card-board#102] [Ideas]
- Allow to give the “Undated” column a custom name
  - example use case: users who consider every undated task a Backlog item would be able to name the “Undated” column accordingly to "Backlog"

**Comment by roovo** [comment]
Great idea -> I've added this to my backlog as issue #105 - making it more general so any pre-named column can have a custom name.  I plan to do this (along with better RTL support) in the next set of tickets to get done after finishing the current release :)

Thanks

**Comment by roovo** [comment]
Just pushed a new release (0.6.7) which allows all the auto-generated column names to be customized :)

**Reply by ItsFullOfCode** [reply]
That's awesome! Thank you so much! :-)
